osf/models/external.py

```diff
@@ -14,7 +14,7 @@
 from requests_oauthlib import OAuth1Session, OAuth2Session
 
 from framework.exceptions import HTTPError, PermissionsError
-from framework.sessions import session
+from framework.sessions import get_session, session
```

Comment:
```suggestion
from framework.sessions import get_session
```

Remove unused `session`